internal/pkg/orgname/orgname.go

```diff
@@ -0,0 +1,24 @@
+package orgname
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/stackitcloud/stackit-cli/internal/pkg/print"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/resourcemanager/client"
+)
+
+// GetOrganizationName returns the name of an organization by its ID.
+func GetOrganizationName(ctx context.Context, p *print.Printer, orgId string) (string, error) {
+	apiClient, err := client.ConfigureClient(p)
+	if err != nil {
+		return "", fmt.Errorf("configure resource manager client: %w", err)
+	}
+	req := apiClient.GetOrganization(ctx, orgId)
+	resp, err := req.Execute()
```

Comment:
This is a new pattern in the CLI I think, right? We could also make this a util of the `resource-manager` service, and initialise the client in the `RunE` of the command which would be more in line with what we were doing until now. What do you think?

Reply:
Making it part of the utils of the `resource-manager` makes sense. I followed the logic of the `projectname` package, that does something similar but for the project name. However, this `GetOrganizationName` would not be used as often, so it may be better to implement in a utils package